app/assets/javascripts/latlong_map.coffee

```diff
@@ -0,0 +1,39 @@
+marker = null
+
+# fallback to PVTA if we can't geocode the incident's location
+PVTA = { lat: 42.105552, lng: -72.596511 }
+
+createMap = (mapSelector, latLng, placeInitialMarker) ->
+  map = new google.maps.Map mapSelector[0], { zoom: 15, center: latLng }
+  if placeInitialMarker
+    placeMarker map, latLng
+  google.maps.event.addListener map, 'click', (event) ->
+    placeMarker map, event.latLng
+    fillLatLngFields event.latLng
+
+fillLatLngFields = (latLng) ->
+  $('#incident_latitude').val latLng.lat()
```

Comment:
Any thought that the map should do anything if these values are changed?

Also, the precision may be a little bit much. I wasn't able to get two points that differed by much more than `0.000_001`